### PR TITLE
DOC: NLG documentation

### PR DESCRIPTION
**Pull request by jaidevd**
NLG documentation split off from https://github.com/gramener/gramex/pull/143

### CODE REVIEW

File: nlg/README.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    YAMLURL: $YAMLURL/nlg/
+    auth:
+      login_url: /$YAMLURL/login
+```
+
+This configuration mounts the app at [nlg/](nlg/):
+
+<div class="example">
+  <a class="example-demo" href="nlg/">NLG</a>
+  <a class="example-src" href="http://github.com/gramener/gramex/blob/master/gramex/apps/guide/nlg/gramex.yaml">Source</a>
```
Comment (pratapvardhan) [inline nlg/README.md]: Could you change the link to  http://github.com/gramexrecipes/gramex-guide/blob/master/nlg/gramex.yaml ?

File: nlg/README.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+```yaml
+import:
+  nlg:
+    path: $GRAMEXAPPS/nlg/gramex.yaml
+    YAMLURL: $YAMLURL/nlg/
+    auth:
+      login_url: /$YAMLURL/login
+```
+
+This configuration mounts the app at [nlg/](nlg/):
```
Comment (pratapvardhan) [inline nlg/README.md]: This is working against which gramex version? I'm not able to run from gramex dev.

Comment (jaidevd) [reply nlg/README.md]: This is supposed to run with [this branch of gramex](https://github.com/gramener/gramex/tree/jd-nlg-doc), the PR for which is at gramener/gramex#143

File: nlg/README.md (hunk 1 above)
Comment (jaidevd) [reply nlg/README.md]: Done.

**Pull request closed without merging**